content/release_notes_manage.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Release Notes: Chef Manage 3.0"
+draft = false
+
+[menu]
+  [menu.infra]
+    title = "Chef Manage"
+    identifier = "chef_infra/release_notes/release_notes_manage.md Chef Manage"
+    parent = "chef_infra/release_notes"
+    weight = 50
++++
+
+[\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/release_notes_manage.md)
+
+Chef Manage provides a web-based user interface to manage Chef Infra nodes and
+other policy objects such as data bags and roles alongside Chef Infra Server.
```

Comment:
@IanMadd I'm not following this change. alongside sounds like "in addition to" Infra Server, but Chef Manage is only an interface to the data on the Infra Server and runs on top of it. knife is the command line tool for modifying data on the Infra Server, and Chef Manage is the web UI for modifying that data.

Reply:
I was trying to clarify the previous text which is a bit clunky. 

> on the same Chef Infra Server as it is installed on.

But maybe 

Chef Manage provides a web-based user interface
that manages Chef Infra nodes and other policy objects,
such as data bags and roles, on the instance of Chef Infra Server that it is installed on.

Reply:
that sounds better.